src/Utilities/PointerVector.hpp

```diff
@@ -306,29 +306,26 @@ struct PointerVector
   using VectorizedAddAssign = std::integral_constant<
       bool,
       blaze::useOptimizedKernels && simdEnabled && VT::simdEnabled &&
-          blaze::IsSIMDCombinable<Type, blaze::ElementType_<VT>>::value &&
-          blaze::HasSIMDAdd<Type, blaze::ElementType_<VT>>::value>;
+                blaze::IsSIMDCombinable<Type, blaze::ElementType_<VT>>::value &&
```

Comment:
is this just formatting changes?  if so why?

Reply:
I ran clang-format on the file and it appears this was changed, that's all.